2022/incoming/fractal-christmas-tree.pod

```diff
@@ -0,0 +1,959 @@
+Author: Will 'the Chill' Braswell <william.braswell@autoparallel.com>
+Title: Fractal Christmas Tree
+Topic: MathPerl::Fractal::ChristmasTree
+
+=encoding utf8
+
+=begin html
+
+<!-- exit HTML <body> section, go back into <head> section, 
```

Comment:
The C<> POD sections are rendered into <code> HTML tags which do very little to differentiate visually from normal text, basically it just changes the font which is very easy to overlook.  My CSS simply highlights the C<> sections for easy visual identification.  I am not against applying this CSS change to all 2023 articles if you like, but for 2022 it can just be my article only.

Reply:
Yes, I did use the PDF preview as it makes the review much easier - thank you for taking the time to provide that.

We can keep this in for now and revisit if subsequently the main site CSS is amended to make the C<> sections more distinct.